lib/board.rb

```diff
@@ -0,0 +1,55 @@
+
+class Board
+  attr_accessor :cells
+
+  def initialize           
+    @cells = Array.new(9, " ")
```

Comment:
@Lani1321 @klovae since our `reset!` method does the same thing. We could just use that here instead.